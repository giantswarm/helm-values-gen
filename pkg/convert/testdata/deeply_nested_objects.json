{
  "$id": "https://example.com/person.schema.json",
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "properties": {
    "address": {
      "properties": {
        "street": {
          "type": "object",
          "properties": {
            "name": {
              "type": "string",
              "default": "Main Street"
            },
            "number": {
              "type": "integer"
            }
          }
        },
        "city": {
          "type": "string"
        }
      },
      "type": "object"
    },
    "age": {
      "type": "integer",
      "default": 21
    },
    "firstName": {
      "type": "string",
      "default": "John"
    },
    "lastName": {
      "type": "string",
      "default": "Doe"
    }
  },
  "type": "object"
}
