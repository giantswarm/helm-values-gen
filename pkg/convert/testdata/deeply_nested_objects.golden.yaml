address:
  street:
    name: Main Street
age: 21
firstName: John
lastName: Doe
